src/backends/lfc.rs

```diff
@@ -23,11 +23,11 @@ impl Backend for LFC {
         let reactor_copy = self.target.main_reactor.clone();
 
         let build_lambda = |main_reactor: &String| -> bool {
-            println!("building main reactor: {}", &main_reactor);
+            println!("--- Building main reactor: {}", &main_reactor);
```

Comment:
we actually maybe need to think about using some proper logging library where we also then can configure verbosity.